crates/oxc_linter/src/snapshots/no_useless_promise_resolve_reject.snap

```diff
@@ -0,0 +1,400 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_useless_promise_resolve_reject
+---
+  ⚠ eslint-plugin-unicorn(no-useless-promise-resolve-reject): Prefer `throw error` over `return Promise.reject(error)`.
+   ╭─[no_useless_promise_resolve_reject.tsx:3:1]
+ 3 │                 if (foo > 4) {
+ 4 │                     return Promise.reject(new Error('🤪'));
+   ·                            ───────────────────────────────
```

Comment:
Does it make sense to add help message explaining why this is useless? Since this is categorized a correctness rule.

Reply:
moved to pedantic, improved the error